packages/react-native/React/Views/UIView+React.m

```diff
@@ -306,9 +307,16 @@ static void updateTransform(RCTPlatformView *view) // [macOS]
     transform = view.reactTransform;
   }
 
+#if !TARGET_OS_OSX // [macOS]
   view.layer.transform = transform;
   // Enable edge antialiasing in rotation, skew, or perspective transforms
   view.layer.allowsEdgeAntialiasing = transform.m12 != 0.0f || transform.m21 != 0.0f || transform.m34 != 0.0f;
+#else // [macOS
+  if ([view isKindOfClass:[RCTView class]]) {
```

Comment:
Would this get better if this PR ever lands? https://github.com/microsoft/react-native-macos/pull/1936

Reply:
This feels like a code smell to me since we're checking in a base class if we're a specific type of subclass. I almost wonder if it's better to copy the implementation of `setTransform3D` to here, but I think that's what the earlier PR I linked does. It feels like the PR above is the more "correct" solution, but this solution unblocks.

Reply:
The PR linked doesn't quite do that; it makes `transform3D` a property of `RCTUIView`, not `NSView`. Perhaps the methods that invoke `updateTransform` (all localized to this file) should be under `RCTUIView` as well.

This isn't an issue in RNCore because `RCTUIView` is a construct exclusive to react-native-macos, and it gets `#define`d as `UIView` when react-native-macos gets compiled for iOS.

Reply:
I think moving logic for `updateTransform` to this file makes sense. I'd rather not have the RCTView -> RCTUIView -> UIView+React -> RCTView dependency cycle.

Reply:
Moving all the logic from `updateTransform` to `RCTUIView` would require additional work based on its callers, but moving the actual meat of the work into `RCTView` is easy enough, as seen in a4c03ca.